src/core/StatusBarItem.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from "vscode";
+import { Graph } from "../typings/Types";
+
+export default class StatusBarItem {
+  private disposables: vscode.Disposable[] = [];
+
+  constructor(private graph: Graph) {
+    graph.extensionContext.subscriptions.push(this);
+  }
+
+  init() {
+    const commandId = "cursorless.showQuickPick";
+    const statusBarItem = vscode.window.createStatusBarItem(
+      vscode.StatusBarAlignment.Right,
+      100
+    );
+    statusBarItem.command = commandId;
+    statusBarItem.text = "$(cursorless-icon) Cursorless";
```

Comment:
I think I might just have the icon without text?  Idk, curious what other people (especially with bigger screens 😄) think cc/ @AndreasArvidsson @auscompgeek

Reply:
I prefer the text. We could probably add the option to hide the text in the menu when you click it?

Reply:
For now, the menu is just showing all user-facing Cursorless commands. Having a command to hide that text feels a little funny to me since it would show up in the command palette all the time. 🤔 

As long as it's not taking up too much space, I have a minor preference for the text as well, since I think it's very easy to miss things in the clutter of icons/text in the status bar.